.gitlab-ci.yml

```diff
@@ -1,11 +1,16 @@
-netlify:
-  type: deploy
+image: python:3-slim-stretch
+stages:
+  - deploy
+
+build and push:
+  stage: deploy
+  before_script:
+     - apt-get update -qq && apt-get install -y -qq wget curl
   script:
-    # Stage
-    - curl -X POST -d {} ${WEBHOOK_STAGE}
-    # Prod
-    - curl -X POST -d {} ${WEBHOOK_PROD}
-    # Dead Man's Snitch Ping
-    - curl ${DMS}
-  only:
```

Comment:
Also without an `only` section this will deploy stage and prod from changes to any branch.